roles/kraken2/tasks/main.yml

```diff
@@ -0,0 +1,29 @@
+---
+
+- name: Create Kraken2 directory
+  file:
+    path: "{{ kraken2_dest }}"
+    state: directory
+
+- name: Download Kraken2
+  get_url:
+    url: "{{ kraken2_download_url }}"
+    dest: "{{ kraken2_src }}"
```

Comment:
I think it would be good to specify the expected checksum for the downloaded file using the `checksum` parameter

Reply:
Also, if this file is very big, it may be good to use e.g. `creates: {{ kraken2_src }}` to avoid this task if the file has already been downloaded. And/or use a tag for this task so that it can be explicitly skipped.

Reply:
How big is very big in this case? The tar.gz is 192 Kb